siege_utilities/survey/models.py

```
@@ -0,0 +1,145 @@
+    count: float                 # weighted or raw count
+    pct: Optional[float] = None  # proportion (0–1); None for non-percent views
+    sig_flag: Optional[str] = None  # column letter if significantly different
+
+
+# ---------------------------------------------------------------------------
+# Chain — one cross-tabulation
+# ---------------------------------------------------------------------------
+
+@dataclass
+class Chain:
+    """One cross-tabulation: row_var × break_vars.
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)